src/rust/collections/shared_ring.rs

```diff
@@ -86,10 +87,11 @@ mod test {
     #[test]
     fn ring_buffer_on_shm_sequential() -> Result<()> {
         let shm_name: String = "shm-test-ring-buffer-serial".to_string();
-        let ring: SharedRingBuffer<u8> = match SharedRingBuffer::<u8>::create(&shm_name, RING_BUFFER_CAPACITY) {
-            Ok(ring) => ring,
-            Err(_) => anyhow::bail!("creating a shared ring buffer should be possible"),
-        };
+        let ring: SharedRingBuffer<RingBuffer<u8>> =
+            match SharedRingBuffer::<RingBuffer<u8>>::create(&shm_name, RING_BUFFER_CAPACITY) {
+                Ok(ring) => ring,
+                Err(_) => anyhow::bail!("creating a shared ring buffer should be possible"),
```

Comment:
Should we log the actual error hear instead?

Reply:
Yes.